rest-helix/src/main/java/com/github/twitch4j/helix/domain/Subscription.java

```diff
@@ -33,10 +33,10 @@
 
     // ID of the subscribed user.
     @NonNull
-    private String userId;
+    private Integer userId;
```

Comment:
You should leave the userId as String. The helix api docs specify the type as string everywhere where the user_id is used.

Parameter | Type | Description
-- | -- | --
user_id | string | ID of the subscribed user.